promgen/static/js/promgen.vue.js

```diff
@@ -273,6 +273,9 @@ app.component("promql-query", {
             if (this.count == 0) return "label label-default";
             return "label label-success";
         },
+        maxAsInt() {
+            return Number.parseInt(this.max);
+        },
```

Comment:
It would be nice to reuse this computed property in line 266:

```
        load: function () {
            return this.count / this.maxAsInt;
        },
```

Reply:
I have seen [496229a](https://github.com/line/promgen/pull/575/commits/496229ab94f8469d55fc6097279811e1fbb0c763). Looks good to me.
Please rebase on master and squash all fixup commits for final review.